File: PRAK402-2210817120004-NOR AINA EVA YANTI.py
bilangan = int(input())
for i in range (1, bilangan) :
    if i % 2 != 0 :
        print(i, end=' ')
print("\n")
while bilangan > 0 :
    if bilangan % 2 == 0 :
        print(bilangan, end=' ')
    bilangan -= 1

File: PRAK402-2210817120004-NOR AINA EVA YANTI.c
#include <stdio.h>

int main()
{
    int i, j, bilangan;
    printf("Batas maksimal bilangan\n");
    scanf("%d", &bilangan);
    for(i=1; i<=bilangan; i++){
        if(i % 2==0){
        }
        else{
            printf("%d", i);
        }
    }
    printf("\n");
    for(j=bilangan; j>=1; j--){
        if(j % 2!=0){
        }
        else{
            printf("%d", j);
        }
    }
}
